applications/functions/doc/README.md

```diff
@@ -0,0 +1,44 @@
+
+# Functions *serverless document processing*
+
+Functions enables dynamic call processing based on customized serverless function and callflow document as input.
+
+## Enabling Functions
+
+Functions has to be started manually by one of these method.
+
+In the case of manual loading, Erlang shell
+```shell
+functions_app:start().
+```
+Or `sup` command in Linux.
+```shell
+sup functions_app start
```

Comment:
Just noting to change this to `sup kapps_controller start_app functions` and to edit `system_config/kapps_controller` to add `functions` to auto-started apps.